worlds/pokemon_emerald/rules.py

```diff
@@ -0,0 +1,1366 @@
+"""
+Logic rule definitions for Pokemon Emerald
+"""
+from BaseClasses import CollectionState, MultiWorld
+from Options import Toggle
+
+from worlds.generic.Rules import add_rule, set_rule
+
+from .options import EliteFourRequirement, NormanRequirement, ExtraBoulders
+from .util import location_name_to_label
+
+
+def _can_cut(state: CollectionState, player: int) -> bool:
```

Comment:
leading _ denotes a private method, which these can never be since they don't have a parent object.

Reply:
I've been using it to say "don't import me" for functions and variables defined in global scope. Pylance/pylint seem to agree with this by not telling me that a function is unused when prefixed with a `_` and not telling me that a function is missing a docstring when prefixed with a `_`.